test/TEST-04-FULL-SYSTEMD/test.sh

```diff
@@ -149,115 +107,24 @@ Type=oneshot
 StandardInput=tty
 StandardOutput=tty
 EOF
-        mkdir -p "$initdir"/etc/systemd/system/testsuite.target.wants
-        ln -fs ../testsuite.service "$initdir"/etc/systemd/system/testsuite.target.wants/testsuite.service
-
-        # make the testsuite the default target
-        systemctl --root="$initdir" set-default testsuite.target
-
-        # install basic tools needed
-        inst_multiple sh bash setsid loadkeys setfont \
-            login sulogin gzip sleep echo mount umount
-        inst_multiple modprobe
-
-        # install libnss_files for login
-        inst_libdir_file "libnss_files*"
 
-        # install dbus and pam
-        inst_multiple -o \
-            /etc/dbus-1/** \
-            /etc/pam.d/** \
-            /etc/security/** \
-            /lib64/security/** \
-            /lib/security/**
-
-        # install dbus socket and service file
-        inst_multiple -o \
-            /usr/lib/systemd/system/dbus.socket \
-            /usr/lib/systemd/system/dbus.service \
-            /usr/lib/systemd/system/dbus-broker.service \
-            /usr/lib/systemd/system/dbus-daemon.service
-
-        (
-            echo "FONT=eurlatgr"
-            echo "KEYMAP=us"
-        ) > "$initrd"/etc/vconsole.conf
-
-        # install basic keyboard maps and fonts
-        for i in \
-            /usr/lib/kbd/consolefonts/eurlatgr* \
-            /usr/lib/kbd/keymaps/{legacy/,/}include/* \
-            /usr/lib/kbd/keymaps/{legacy/,/}i386/include/* \
-            /usr/lib/kbd/keymaps/{legacy/,/}i386/qwerty/us.*; do
-            [[ -f $i ]] || continue
-            inst "$i"
-        done
-
-        # some basic terminfo files
-        for _terminfodir in /lib/terminfo /etc/terminfo /usr/share/terminfo; do
-            [ -f ${_terminfodir}/l/linux ] && break
-        done
-        inst_multiple -o ${_terminfodir}/l/linux
-
-        # softlink mtab
-        ln -fs /proc/self/mounts "$initdir"/etc/mtab
```

Comment:
Maybe this would avoid https://github.com/dracutdevs/dracut/actions/runs/3566589541/jobs/5993228395#step:4:447 , not happening before this PR.